TUTORIAL-TOOL.md

```diff
@@ -334,8 +343,8 @@ $ zip -r my-indoor-map.zip .
 
 Troubleshooting
```

Comment:
As per comment in README.md this section can be factored out and replaced by a link to the general troubleshooting section.